client/src/store/modules/profile.js

```diff
@@ -126,6 +161,15 @@ export default {
     },
     setPreferences (state, data) {
       state.myProfile.preferences = data
+    },
+    isLoading (state) {
+      return state.isLoading
+    },
+    deleteError (state) {
+      state.errors = state.errors.filter(el => el.error !== "You have been blocked by this user")
```

Comment:
Error are usually translatable, for future try using error codes (numbers aka 519) so their texts could be translated without interfering with their logic.